src/ppx/application.cpp

```diff
@@ -1174,7 +1148,12 @@ int Application::Run(int argc, char** argv)
         return EXIT_FAILURE;
     }
 
+    // Call config.
```

Comment:
Run() seems to be a massive function. Perhaps not in this PR, but maybe we should extract pretty much line 1145-1233 into it's own function.

Reply:
make sense, i can do another PR to clean this up once this PR is submitted